app/src/main/java/org/fossasia/openevent/app/data/cache/ObjectCache.java

```diff
@@ -0,0 +1,34 @@
+package org.fossasia.openevent.app.data.cache;
+
+
+import android.support.v4.util.LruCache;
+
+import org.fossasia.openevent.app.contract.model.CacheModel;
+
+public class ObjectCache implements CacheModel {
+
+    private static ObjectCache instance;
```

Comment:
And as it is not dependent on android, we need not mock its behaviour, we can just use actual object and see if it works